crates/ark-contracts/arkchain/src/orderbook.cairo

```diff
@@ -402,7 +398,69 @@ mod orderbook {
     // *************************************************************************
     #[generate_trait]
     impl InternalFunctions of InternalFunctionsTrait {
-        /// Fulfill order
+        /// Fulfill auction order
+        ///
+        /// # Arguments
+        /// * `fulfill_info` - The execution info of the order.
+        /// * `order_type` - The type of the order.
+        ///
+        fn _fulfill_auction_order(
+            ref self: ContractState, fulfill_info: FulfillInfo, order: OrderV1
+        ) {
+            assert(
+                order.offerer == fulfill_info.fulfiller, orderbook_errors::ORDER_NOT_SAME_OFFERER
+            );
+            let AUCTION_ACCEPTING_TIME: u64 = 86400;
+            assert(
+                order.end_date + AUCTION_ACCEPTING_TIME > starknet::get_block_timestamp(),
```

Comment:
Don't we want to test the `end_date` of the order that can be moved by new offers? The `end_date` must come from the storage of the auction end date, and not the order itself which is never modified.

Do you prefer 24h instead of 48h? Can be good to have a comment explaining this time.

Reply:
oh true i used the wrong date here